docs/language-server/protocol-language-server.md

```diff
@@ -376,6 +376,11 @@ interface ExpressionUpdate {
    *  - array with multiple values represents an intersetion type
    */
   type: string[];
+  /**
+   * The list of types this expression can be converted to.
+   */
+  hiddenType: string[];
+  expressionType?: ExpressionType;
```

Comment:
What does the new `expressionType` field represent?

Reply:
Good catch, forgot to remove it